pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/JSONParser.groovy

```diff
@@ -113,9 +113,6 @@ class JSONParser implements Parser {
                 case 'parameters':
                     pipelineDef.parameters = parseBuildParameters(pipelineJson.getJSONObject("parameters"))
                     break
-                case 'wrappers':
```

Comment:
Nah - we don't really care about that for the JSON at this point. Plus, `wrappers` would actually fail at schema validation time, before we even got this far.